versions.tf

```diff
@@ -1,14 +1,14 @@
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = ">= 1.11.0"

```

Comment:
I would advise against `>= 1.11.0` because there is nothing in this module that requires features unique to Terraform v1.11. The [Terraform compatibility promises](https://developer.hashicorp.com/terraform/language/v1-compatibility-promises) ensure that Terraform modules written for v1.0 will always work throughout the v1.x release cycle.

Additionally, because our internal pipelines are still using TF 1.8 or 1.9, requiring v1.11 would be problematic. 

Therefore, I advise updating this to `>= 1.0` instead.